### ISSUE TITLE
Error in seq.int(0, to0 - from, by) : wrong sign in 'by' argument

### ISSUE THREAD

**Issue by rlumor** [bcgov/fasstr#81]
I have been using the fill_missing_date functionality within the package to fill missing data values. I have  made sure no event date  occurs before a start date within my dataset but I keep getting the same error. Will there be any other reason why this error will occur?

**Comment by rlumor** [comment]
mydata_at3 <- mutate(mydata_at2,
                      make_model_new=str_c(make_name,model_name,sep='_')
                     )%>%
              fasstr::fill_missing_dates(
                      dates = eventdate,
                      values=visits,
                      groups=make_model_new,
                      water_year_start = mths,
                      fill_end_years=FALSE
                     )

**Comment by jongoetz** [comment]
That's a good question, @rlumor! Is your data set a daily time series with dates formatted yyyy-mm-dd (%Y-%m-%d) consistently throughout in all of the groups? 
Is your mths variable for water_year_start different than 1? water_year_start isn't required if your fill_end_years=FALSE, but it may help me find out where an error may be occurring.

Within the fill_missing* function, the dates are first checked and formatted to yyyy-mm-dd (so if the Y, m and d are in a different order they may be translated incorrectly). Then for each group the minimum date and the maximum date are used to set the full sequence (seq() ) of dates, so the order of the dates should not matter (I think). I tried a few things to get it to replicate your error to no suffice. Any examples of your data may help!

**Comment by rlumor** [comment]
@jongoetz thanks for the response!. Yes to all your questions :). This is what I realised yesterday after I posted by question, there are date sequences where the end date and start date is the same so when seq() is applied i think its generates the error because those dates are not a sequence :). see the example below for Buick, Ferrari and Tesla. What i ended up doing was deleting rows which meet this criteria 
![image](https://user-images.githubusercontent.com/14334370/118817222-fde68600-b880-11eb-943b-f2b12f95faf9.png)
